static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -1,13 +1,17 @@
 $hosted-stripe: 28px;
+$hosted-video-width: 960px;
+$hosted-video-height: 540px;
 
 @mixin hosted-fade-in {
+    visibility: visible;
     opacity: 1;
-    transition: opacity 1s linear;
+    transition: opacity 1s linear, visibility 0s;
```

Comment:
Which ARIA live attribute do you have in mind? I can use the `aria-live="polite"` on the header, so the screen reader will speak it's value whenever the user is idle.

And yes, I want to make the header completely invisible, because after changing only the `opacity: 0`, the text layer is still there and it's impossible to click on the 'play' overlay.